### PR TITLE
Adding subtitle to half-width and info-unit molecules

### PR DESCRIPTION
**Pull request by sebworks**
Modified the half-width and info-unit molecules to allow sub-headings. This is necessary for the 
http://www.consumerfinance.gov/about-us/careers/ page. 

## Changes

- Modified `cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html`  and `cfgov/jinja2/v1/_includes/molecules/info-unit.html` to allow subtitles.

## Testing

- This will require modifications to wagtail before you can view the subtitle. 

## Screenshot

screen shot 2016-08-04 at 11 55 51 am

## Review

- @willbarton 
- @chosak 
- @richaagarwal 
- @virginiacc 
- @ajbush 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

…ding for Careers pages

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html [hunk 1]
```diff
@@ -8,25 +8,44 @@
...
-   value:               Object defined from a StreamField block.
+   value:                  Object defined from a StreamField block.
 
-   value.heading:       (Optional) A string containing title of module.
+   value.heading:          (Optional) A string containing title of module.
 
-   value.body:          A string containing the content to display below the
-                        heading (typically a paragraph).
+   value.sub_heading:      (Optional) A string containing subtitle
+                           of module.
```
Comment (richaagarwal) [inline cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html]: Is there a reason `value.body` and `value.links` are removed here?

**Comment by richaagarwal** [comment]
@sebworks Do you need one of the BEWDs to update the model?  I noticed that `HalfWidthLinkBlob` hasn't been updated:
```
class HalfWidthLinkBlob(blocks.StructBlock):
    heading = blocks.CharBlock(required=False)
    body = blocks.RichTextBlock(blank=True, required=False)
    links = blocks.ListBlock(atoms.Hyperlink(), required=False)

    class Meta:
        icon = 'link'
        template = '_includes/molecules/half-width-link-blob.html'

```

**Comment by sebworks** [comment]
@richaagarwal, yes, the BEWDS will need to update the model to accomodate the new fields.

**Comment by virginiacc** [comment]
The subtitle code looks good & works in local testing. 👍 Will the model updates happen after this is merged?

**Comment by richaagarwal** [comment]
@sebworks I can make make commits directly to this PR if you'd like.  Unless there's a reason to merge this in without the backend component.

**Comment by sebworks** [comment]
@richaagarwal, I've modified the model and added the migration. Please let me know if I've broken the internet.

**Comment by sebworks** [comment]
@richaagarwal, please review when you have a chance.

**Comment by richaagarwal** [comment]
Ack sorry!  Unrollme did something weird to my github notifications so I totally haven't been seeing any of these :(  Reviewing now.

File: cfgov/v1/atomic_elements/molecules.py [hunk 2]
```diff
@@ -13,6 +13,7 @@ def isRequired(field_name):
 
 class HalfWidthLinkBlob(blocks.StructBlock):
     heading = blocks.CharBlock(required=False)
+    sub_heading  = blocks.CharBlock(required=False)
```
Comment (richaagarwal) [inline cfgov/v1/atomic_elements/molecules.py]: Minor thing but there appear to be two spaces before the assignment here.

**Comment by richaagarwal** [comment]
Where does the definition of info-unit live? I don't see it in molecules anywhere, just curious where it would inherit the new subheading you added.

**Comment by sebworks** [comment]
I don't think we have documentation for the `m-info-unit` on the wiki because we don't use it directly in Wagtail. It gets utilized by other Atomic components though.

**Comment by sebworks** [comment]
@richaagarwal, this issue might be helpful:  /GHE/flapjack/Modules-V1/issues/4.

**Comment by richaagarwal** [comment]
Ooh, I see what is going on now. 👍

**Pull request merged**